solr/solr-ref-guide/src/schema-api.adoc

```diff
@@ -112,7 +93,7 @@ curl -X POST -H 'Content-type:application/json' --data-binary '{
      "name":"sell_by",
      "type":"pdate",
      "stored":true }
-}' http://localhost:8983/solr/gettingstarted/schema
+}' http://localhost:8983/solr/techproducts/schema
```

Comment:
[+1] Good thought to standardize on techproducts here!

Reply:
Meh, I'm frankly -0 on changing every example to use techproducts. Sure it's an example we've always shipped, but I also think it's very limited. Not that I have any suggestions for alternatives, I just think there are 100 other things I'd rather do in the docs than spend time on that.

Reply:
I end up using techproducts a lot to test things, and a way to learn about the apis.    As long as you are okay (i.e -0) with it, then I'll leave it with techproducts.  Thanks for the comment Cassandra!